matlab/schemas/+vis/FancyBar.m

```diff
@@ -1,18 +1,20 @@
  %{
-vis.FancyBar (manual) # pink noise with periods of motion and orientation$
+vis.FancyBar (manual) # Moving Bar stimulus that keeps the size and speed
```

Comment:
compensates for the flat screen. Things on the periphery and center do not have the same size/speed unless corrected.